src/utils/masonryDetails.js

```diff
@@ -1,97 +1,51 @@
-const masonryDetails = {
-  accordion: {
-    'Focus Expectations': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Semantic Elements': {
-      rowStart: 1,
-      numberOfRows: 1,
-    },
-    'Aria Attributes': {
-      rowStart: 'auto',
-      numberOfRows: 1,
-    },
-    'Keyboard Interactions': {
-      rowStart: 2,
-      numberOfRows: 1,
-    },
-    'Additional Resources': {
-      rowStart: 2,
-      numberOfRows: 1,
-    },
+import { marked } from 'marked';
+
+export const rowStart = (markdown) => {
+  const htmlFromMd = marked.parse(markdown.replace(/\\/g, ''), {
+    breaks: true,
+  });
```

Comment:
I hope you don't mind if I comment here, trying to re-explain it and make sure I understand! 

So we have new props that get passed to the Specification block component. My first question is: the `rowStart` and `masonryDetails` are both exported. Are they imported anywhere?

Then there's some checking to see how long the markdown from `RichText` is. What is the significance of returning 1, 2 or 3? I assume it has something to do with the placement of those blocks. 

At line 22, that `specificationBlocks` const is a list of all the specifications we have in Airtable, and is assigned to each component in the `masonryDetails` const. 

(so sorry all my ignorance is showing!)

Reply:
Hey there, as you know from our pairing I still have some ignorance surrounding it too! :) It's rather complex, but I'd like to revisit this after Monday demo. I think after that, it will make much better sense to both of us!